README.md

```diff
@@ -37,6 +37,15 @@ Run locally using SBT:
 
 ` env SBT_OPTS="-Dconfig.file={path to custom config file}" sbt "runMain tech.cryptonomic.conseil.Conseil"`
 
+Or you can use the predefined tasks:
+- for Conseil:
+`sbt -Dconfig.file="{path to custom config file}" runConseil`
+- for Lorre 
+`sbt -Dconfig.file="{path to custom config file}" "runLorre alphanet"`
+- for schema generation
+`sbt -Dconfig.file="{path to custom config file}" genSchema`
+
+
```

Comment:
I would explicitly state here that using the predefined tasks you get a custom-tailored vm launch configuration that's best suited for the actual task

Reply:
ok, quoted you in the docs 😄